source/linear-algebra/source/03-AT/03.ptx

```diff
@@ -243,7 +244,9 @@ Which of these subspaces of <m>\IR^3</m> describes the set of all vectors that a
     <statement>
         <p>
 Let <m>T: V \rightarrow W</m> be a linear transformation.
-The <term>image</term> of <m>T</m> is an important subspace of <m>W</m> defined by
+The <term>image</term> of <m>T</m>
+(also known as the <term>column space</term> of <m>T</m>)
```

Comment:
```suggestion
```

Reply:
I don't see this change in the preview, but I'm ok with it. I would also like to put an index tag here, since the index only takes it to the later section. I'm not sure if I can do that on an existing pull request?